dashboard/test/models/ability_test.rb

```diff
@@ -738,10 +738,28 @@ class AbilityTest < ActiveSupport::TestCase
     refute Ability.new(student_1).can? :view_as_user_for_code_review, @login_required_script_level, student_2
   end
 
-  test 'workshop admin can update scholarship info' do
+  test 'workshop admins can update scholarship info' do
     workshop_admin = create :workshop_admin
     pd_enrollment = create :pd_enrollment
 
     assert Ability.new(workshop_admin).can? :update_scholarship_info, pd_enrollment
   end
+
+  test 'workshop admins can see, update, and delete incomplete applications' do
```

Comment:
Thanks for adding these!

Reply:
+1, I find these super helpful in tracking down whether any problems that come up are within ability.rb or within the controllers.